clients/banking/src/components/TransactionDetail.tsx

```diff
@@ -457,10 +457,54 @@ export const TransactionDetail = ({ transaction, large }: Props) => {
             )
             .with(
               { __typename: "SEPADirectDebitTransaction" },
-              ({ mandate, creditor, reference }) => (
+              ({ mandate, creditor, reference, reservedAmount, reservedAmountReleasedAt }) => (
                 <ReadOnlyFieldList>
                   {bookingDateTime}
 
+                  {isNotNullish(reservedAmount) && (
+                    <LakeLabel
+                      type="viewSmall"
+                      label={t("transaction.reservedAmount")}
+                      render={() => (
+                        <LakeText variant="regular" color={colors.gray[900]}>
+                          {(transaction.side === "Debit" ? "-" : "+") +
+                            formatCurrency(Number(reservedAmount.value), reservedAmount.currency)}
+                        </LakeText>
```

Comment:
probably stupid question here : are you sure the signe is always in front of the amount, in every locales and currencies ?

couldn't it be more safe to do something like this : 

formatCurrency(Number(transaction.side === "Debit" ? "-" : "+") + reservedAmount.value), reservedAmount.currency)

Reply:
It could be better, let's discuss with the team